api/system-status/read

```diff
@@ -32,7 +32,45 @@ function readMemory()
     return $fields; 
 }
 
-function readUptime() {
+function readDisk()
+{
+    $out = array();
+    $ret = array();
+    exec('/bin/df -P /', $out);
+    # Filesystem Size  Used Avail Use% Mount
+    for ($i=0; $i<count($out); $i++) {
+        if ($i == 0) {
+            continue;
+        }
+        $tmp = explode(" ", preg_replace('/\s+/', ' ', $out[$i]));
+        // skip fs ($tmp[0]) and perc_used ($tmp[4])
+        if ($tmp[5] == "/") {
+          $tmp[5] = "root";
+        }
+        $ret[$tmp[5]] = array('total' => intval($tmp[1]), 'used' => intval($tmp[2]), 'free' => intval($tmp[3]));
+    }
+    return $ret; 
+}
+
+function readRaid()
+{
+    $out = array();
+    $ret = array();
+    $out = json_decode(exec('/usr/libexec/nethserver/raid-status'), true);
```

Comment:
If you need just to report if there are any errors, you do not need to parse the output: just check the exit value.
If the exit value is different than 0, md is are errors.

Reply:
This is new for me ^-^

The output of an OK RAID is this:
```
[root@fw ~]# /usr/libexec/nethserver/raid-status
{"results":[{"md":"md1","tot_devs":"2","spare_devs":"none","active_devs":"sdb1,sda1","failed_devs":"none","ok_devs":"2","level":"1","result":"ok"},{"md":"md2","tot_devs":"2","spare_devs":"none","active_devs":"sdb2,sda2","failed_devs":"none","ok_devs":"2","level":"1","result":"ok"}]}
```

How I have to change my code?

Reply:
This is the magic line on the raid script: https://github.com/NethServer/nethserver-base/blob/master/root/usr/libexec/nethserver/raid-status#L100

To use it, just change your code like (I didn't test it):
```php
exec('/usr/libexec/nethserver/raid-status', $out, $exit_code);
if ($exit_code == 0) {
        $ret['status'] = 'ok';
} else {
        $ret['status'] = 'error';
}
return $ret
```

Reply:
I made some changes to hide RAID status if RAID is not configured. Can you check if it's ok for you? It should works fine.